README.md

```diff
@@ -1 +1,16 @@
+# Dumb things!
+
 A dumb repo with dumb code.
+
+A dumb subheader!
```

Comment:
Headings should use atx<br>Don’t add a trailing `!` to headings<br>Don’t use multiple top level headings (5:1)